python/ray/experimental/serve/examples/echo.py

```diff
@@ -0,0 +1,28 @@
+"""
+Example service that prints out http context
+"""
+
+import time
+from pprint import pprint
+
+import requests
+
+import ray.experimental.serve as srv
+
+
+def echo(context):
```

Comment:
I would include docs that explain what the function arg list needs to look like (i.e. specify what this context is, and that it is necessary)

Reply:
Preprocessing will be in next PR. Thanks for noticing.